packages/gnome_tweaks.rb

```diff
@@ -0,0 +1,40 @@
+# Adapted from Arch Linux gnome-tweaks PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/gnome-tweaks/trunk/PKGBUILD
+
+require 'package'
+
+class Gnome_tweaks < Package
+  description 'Graphical interface for advanced GNOME 3 settings Tweak Tool'
+  @_ver = '40.beta'
+  version @_ver
+  compatibility 'x86_64 , armv7l'
+  source_url "https://gitlab.gnome.org/GNOME/gnome-tweaks/-/archive/40.beta/gnome-tweaks-#{@_ver}.tar.bz2"
```

Comment:
`source_url "https://gitlab.gnome.org/GNOME/gnome-tweaks/-/archive/#{@_ver}/gnome-tweaks-#{@_ver}.tar.bz2"`?

Reply:
Sure, but this might move to a 40/40.something at some point...